acceptance-tests/runner/nightwatch.conf.js

```diff
@@ -0,0 +1,65 @@
+module.exports = (function (settings) {
```

Comment:
This file already exists in the "acceptance-tests" dir. Can one of them be deleted?